pkg/antctl/antctl.go

```diff
@@ -160,9 +165,10 @@ var CommandList = &commandList{
 			transformedResponse: reflect.TypeOf(agentinfo.AntreaAgentInfoResponse{}),
 		},
 		{
-			use:   "podinterface",
-			short: "Print Pod's network interface information",
-			long:  "Print information about the network interface(s) created by the Antrea agent for the specified Pod.",
+			use:     "podinterface",
+			aliases: []string{"podinterfaces", "pod-interface", "pod-interfaces"},
```

Comment:
don't think we want different styles of commands, I think we should make commands style consistent, singular, plural and short name are enough.

Reply:
Removed "pod-interface" and "pod-interfaces", added "pi".